controller/admin.go

```diff
@@ -188,6 +199,187 @@ func AdminAssignmentGET(w http.ResponseWriter, r *http.Request) {
 	v.Render(w)
 }
 
+// AdminAssignmentCreateGET handles GET-request from /admin/assigment/create
+func AdminAssignmentCreateGET(w http.ResponseWriter, r *http.Request) {
+	//check that user is a teacher
+	if !session.IsTeacher(r) { //not a teacher, error 401
+		ErrorHandler(w, r, http.StatusUnauthorized)
+		return
+	}
+
+	var subRepo = model.SubmissionRepository{}
+	submissions, err := subRepo.GetAll()
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+
+	v := view.New(r)
+	v.Name = "admin/assignment/create"
+
+	v.Vars["Courses"] = []struct { // TODO (Svein): use database
```

Comment:
Use finished function to get courses
```suggestion

	v.Vars["Courses"] = model.GetCoursesToUser(session.GetUserFromSession(r).ID)
```